package/docker/bahmni-reports/template/bahmni-reports.properties.template

```diff
@@ -10,8 +10,8 @@ reports.password=${REPORTS_DB_PASSWORD}
 openmrs.service.rootUrl=http://${OPENMRS_HOST}:${OPENMRS_PORT}/openmrs/ws/rest/v1
 openmrs.service.user=reports-user
 openmrs.service.password=Admin123
-openmrs.connectionTimeoutInMilliseconds=30000
-openmrs.replyTimeoutInMilliseconds=120000
+openmrs.connectionTimeoutInMilliseconds=600000
```

Comment:
@daviemukungi this property need not to be changed. Only read timeout need to be updated.

Reply:
This has been resolved @sivareddyp